### ISSUE TITLE
[Bug] Error when run npm test: stdout maxBuffer exceeded

### ISSUE THREAD

**Issue by beenotung** [DefinitelyTyped/DefinitelyTyped#36059]
When I run `npm test` on the root of DefinitelyTyped, I got below error in the console:
```
$ npm test

> definitely-typed@0.0.3 test /home/beenotung/workspace/github.com/beenotung/DefinitelyTyped
> node node_modules/types-publisher/bin/tester/test.js --run-from-definitely-typed

Clean data
Clean logs
Clean output
Using local Definitely Typed at /home/beenotung/workspace/github.com/beenotung/DefinitelyTyped.
Parsing definitions...
Found 5881 packages.
Parsing in parallel...
Running: git rev-parse --verify master
96ecf8642b80681d2a1890dedd2b3569095df725
Running: git diff master --name-status
Error: Error: stdout maxBuffer exceeded
    at Socket.onChildStdout (child_process.js:329:14)
    at emitOne (events.js:116:13)
    at Socket.emit (events.js:211:7)
    at addChunk (_stream_readable.js:263:12)
    at readableAddChunk (_stream_readable.js:246:13)
    at Socket.Readable.push (_stream_readable.js:208:10)
    at Pipe.onread (net.js:601:20)

    at Object.execAndThrowErrors (/home/beenotung/workspace/github.com/beenotung/DefinitelyTyped/node_modules/.github.com/Microsoft/types-publisher/51db099db4d654ab4c12c46035541b53b2b00801/node_modules/types-publisher/src/util/util.ts:166:15)
    at <anonymous>
    at process._tickCallback (internal/process/next_tick.js:189:7)
 ERROR  Test failed. See above for more details.
```

Not sure who to mention, but it seems below authors are knowledgeable on the script setting in package.json:
@Flarna @sandersn

**Comment by sandersn** [comment]
@beenotung how long has it been since you merged master to the branch you're working on? Does that fix the problem? 

In the meantime I'll go see how big stdout maxBuffer is by default.

**Comment by sandersn** [comment]
The source code actually has a comment saying that this has been a problem before and links [to a previous comment](https://github.com/DefinitelyTyped/DefinitelyTyped/pull/26545#issuecomment-402274021). Unfortunately, the page won't load for me so I don't know the details.

In any case, the correct fix may be to increase the maxBuffer size by 4x one more time.

**Comment by hlovdal** [comment]
Duplicate of #35740.

**Comment by orta** [comment]
Hi thread, we're moving DefinitelyTyped to use [GitHub Discussions](https://github.com/DefinitelyTyped/DefinitelyTyped/issues/53377) for conversations the `@types` modules in DefinitelyTyped.

To help with the transition, we're closing all issues which haven't had activity in the last 6 months, which includes this issue. If you think closing this issue is a mistake, please pop into the [TypeScript Community Discord](https://discord.gg/typescript) and mention the issue in the `definitely-typed` channel.
